opsgenie/resource_opsgenie_team_test.go

```diff
@@ -156,6 +258,22 @@ resource "opsgenie_team" "test" {
 `, rString)
 }
 
+func testAccOpsGenieTeam_basicNoMember(randomUser, rString string) string {
+	return fmt.Sprintf(`
+resource "opsgenie_team" "test" {
+  name           = "genieteam-%s"
+  description    = "This team deals with all the things"
+  ignore_members = false
```

Comment:
can. you update this field as well

Reply:
otherwise our e2e test will not pass :)

Reply:
```
TestAccOpsGenieTeam_basicNoMember
  === RUN   TestAccOpsGenieTeam_basicNoMember
--- FAIL: TestAccOpsGenieTeam_basicNoMember (2.52s)
    testing.go:569: Step 0 error: After applying this step and refreshing, the plan was not empty:
        
        DIFF:
        
        UPDATE: opsgenie_team.test
          description:    "This team deals with all the things" => "This team deals with all the things"
          id:             "1616fef5-4fb2-43b3-baf0-deb36a7fc9c9" => "1616fef5-4fb2-43b3-baf0-deb36a7fc9c9"
          ignore_members: "false" => "false"
          member.#:       "1" => "0"
          member.0.id:    "18ab82fa-1461-4e19-80f3-ed5caf85cd6c" => ""
          member.0.role:  "user" => ""
          name:           "genieteam-mn6lfl" => "genieteam-mn6lfl"
        
        
        
        STATE:
        
        opsgenie_team.test:
          ID = 1616fef5-4fb2-43b3-baf0-deb36a7fc9c9
          provider = provider.opsgenie
          description = This team deals with all the things
          ignore_members = false
          member.# = 1
          member.0.id = 18ab82fa-1461-4e19-80f3-ed5caf85cd6c
          member.0.role = user
          name = genieteam-mn6lfl
        
          Dependencies:
            opsgenie_user.test
        opsgenie_user.test:
          ID = 18ab82fa-1461-4e19-80f3-ed5caf85cd6c
          provider = provider.opsgenie
          full_name = Acceptance Test User
          locale = en_US
          role = User
          timezone = America/New_York
          username = genietest-z7m04a@opsgenie.com
FAIL
  INFO[2020-03-19T10:13:28.562510501Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:28.569737798Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:28.570630182Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:29.660732008Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:29.71641125Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.05982477Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.126664758Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.196179068Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.336264972Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.346678315Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.347505211Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.415075811Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.485343337Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.500183214Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.501104884Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.553893579Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.621665678Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.622671852Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.843289972Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
INFO[2020-03-19T10:13:30.968690084Z] Client is configured with ApiUrl: api.opsgenie.com, LogLevel: info, RetryMaxCount: 10 
```

Reply:
fixed. my n-th opsgenie test account has expired so I did not re-restarted e2e tests.